src/network/networkprotocol.h

```diff
@@ -987,12 +987,12 @@ enum NetProtoCompressionMode {
 };
 
 const static std::string accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
-	"Invalid password",
+	"The password is invalid.",
```

Comment:
Your*

Reply:
The original says the same clearly in fewer words -1